example_byteslice_test.go

```diff
@@ -11,7 +14,18 @@ func ExampleReverse() {
 
 func ExampleLPad() {
 	data := []byte{0x55, 0xDA, 0xBA}
-	
+
 	fmt.Printf("%x\n", LPad(data, 5, 0x22))
 	// Output: 222255daba
 }
+
+func ExampleUnset() {
+	data1 := []byte{0x00, 0x00}
+	data2 := []byte{0x00}
+	val, err := Unset(data1, data2)
+	if err == nil || val != nil {
+		log.Println("Unset with two byte slices of different size needs to return an error and no value")
```

Comment:
Don't log the same text as the testcase. It's a example, not a test.
You need to show a possible usage in a example.